user-management.yaml

```diff
@@ -388,6 +394,66 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenericError'
+         
+  /users/{username}/force:            
+    delete:
+      tags:
+      - "User Management"
+      summary: "Force deletes a user"
+      description: "Force-deletes a user (when invoked by an administrator). Removes the user completely. This might lead to errors if invoked on lecturers with existing courses."
```

Comment:
```suggestion
      description: "Force-deletes a user (when invoked by an administrator). Removes the user completely. This might lead to errors, e.g. if invoked on lecturers with existing courses."
```